components/config/src/config/languages.rs

```diff
@@ -14,11 +14,11 @@ pub struct LanguageOptions {
     pub title: Option<String>,
     /// Description of the site. Defaults to None
     pub description: Option<String>,
-    /// Whether to generate a feed for that language, defaults to `false`
-    pub generate_feed: bool,
-    /// The filename to use for feeds. Used to find the template, too.
-    /// Defaults to "atom.xml", with "rss.xml" also having a template provided out of the box.
-    pub feed_filename: String,
+    /// Whether to generate feeds for that language, defaults to `false`
+    pub generate_feeds: bool,
```

Comment:
this will be a breaking change, imo should be marked `#[serde(alias = "generate_feed")]` for backwards compat